pombola/writeinpublic/views.py

```diff
@@ -218,11 +212,32 @@ def get_template_names(self):
         return [self.adapter.get_templates()[self.steps.current]]
 
     def get_context_data(self, form, **kwargs):
+        step = kwargs.get('step')
         context = super(WriteInPublicNewMessage, self).get_context_data(form=form, **kwargs)
         context['message'] = self.get_cleaned_data_for_step('draft')
         recipients = self.get_cleaned_data_for_step('recipients')
+
         if recipients is not None:
-            context['persons'] = recipients.get('persons')
+            persons = recipients.get('persons')
+            if step == 'draft':
+                # On the draft step, check if all recipients are contactable
+                # in WriteInPublic
+                not_contactable = []
+                contactable = []
+                for person in persons:
+                    try:
+                        if not self.client.get_person_is_contactable(person):
+                            not_contactable.append(person)
+                        else:
```

Comment:
is there already a test for this? I'm happy just to have one success case test, the rest can be handled with regression tests as errors come in

Reply:
Yup, there is this test: https://github.com/OpenUpSA/pombola/pull/47/files#diff-9aa30a36f8728d1db5955b769cc6ac3eR224